pkg/oc/cli/admin/diagnostics/diagnostics/cluster/network/in_pod/util/runtime.go

```diff
@@ -0,0 +1,155 @@
+package util
```

Comment:
I think I would have gone with an interface/implementations approach here, but I also think we should be using a shared library for container abstractions (which @smarterclayton already nixed in the form of a kubelet dependency)... so I'm not going to argue much with the implementation design. @smarterclayton, is this better than allowing the kubelet dep? Is there some other runtime interface outside the kubelet we could be using?